cmd/mixpanel_to_redshift.go

```diff
@@ -16,15 +16,21 @@ import (
 
 var (
 	// TODO: include flag validation
-	awsRegion          = env.MustGet("AWS_REGION")
 	jsonpathsFile      = flag.String("jsonpathsfile", "", "s3 file with jsonpaths data.")
 	mixpanelEvents     = flag.String("mixpanelevents", "", "Comma separated values of events to be exported.")
 	mixpanelExportDate = flag.String("exportdate",
 		time.Now().AddDate(0, 0, -1).Format("2006-01-02"),
 		"Date in YYYY-MM-DD format. Defaults to yesterday.")
-	mixpanelExportDir  = flag.String("exportdir", "", "Directory to store the exported mixpanel data.")
-	redshiftSchema     = flag.String("redshiftschema", "public", "Schema with the redshift table.")
-	redshiftTable      = flag.String("redshifttable", "", "Name of the redshift table.")
+	mixpanelExportDir = flag.String("exportdir", "", "Directory to store the exported mixpanel data.")
+	host              = flag.String("host", "", "Address of the redshift host")
+	port              = flag.String("port", "5439", "Port of the redshift host")
+	db                = flag.String("database", "", "Redshift database to connect to")
+	user              = flag.String("user", "", "Redshift user to connect as")
+	schema            = flag.String("schema", "public", "Schema with the redshift table.")
+	table             = flag.String("table", "", "Name of the redshift table.")
```

Comment:
Nit: password, here (I'd also accept `pass`). I keep reading this as something to do with the working directory. :(